shared/common-adapters/dropdown.tsx

```diff
@@ -112,14 +112,15 @@ class Dropdown<N extends React.ReactNode> extends React.Component<Props<N> & Ove
 }
 
 type InlineDropdownProps = {
-  label: string
+  label: React.ReactNode
   onPress: () => void
-  type: 'Body' | 'BodySmall'
+  type: 'Body' | 'BodySmall' | 'BodySmallSemibold'
+  style?: Styles.StylesCrossPlatform
 }
 
 export const InlineDropdown = (props: InlineDropdownProps) => {
   const selected = (
-    <Box2 direction="horizontal" key={props.label} style={styles.inlineSelected}>
```

Comment:
presumably this had a key for a reason.. are you sure you can just remove it?

Reply:
Checked, it is not and was not needed.